### ISSUE TITLE
Failure using jwt.decode

### ISSUE THREAD

**Issue by hir12111** [buildbot/buildbot#3361]
Last version of jwt raises an exception if algorithms parameter is not used (commit [11f30c4050a11b6398d38f505578c9dabeba6c78](https://github.com/jpadilla/pyjwt/commit/11f30c4050a11b6398d38f505578c9dabeba6c78))
It is expecting a list of allowed algorithms to decode the token. 
The test using it is test_www_service.TestBuildbotSite:test_updateSession, it should use a list with the algorithm used when encoding.

```
[ERROR]
Traceback (most recent call last):
  File "/home/hir12111/dls-buildbot/prefix/lib/python2.7/site-packages/twisted/trial/_synctest.py", line 1049, in run
    warnings.warn_explicit(**w)
exceptions.DeprecationWarning: It is strongly recommended that you pass in a value for the "algorithms" argument when calling decode(). This argument will be mandatory in a future version.

buildbot.test.unit.test_www_service.TestBuildbotSite.test_updateSession
buildbot.test.unit.test_www_service.TestBuildbotSite.test_updateSession
```

**Comment by tardyp** [comment]
can you please be a bit more explicit on what is the error?
is it an issue with a latest jwt version?
what do the test pass now?

**Comment by hir12111** [comment]
sorry about that, I added some more details

**Comment by tardyp** [comment]
thanks for details. looks like this is related to https://github.com/jpadilla/pyjwt/issues/258

**Comment by seankelly** [comment]
Fixed in #3362.
